### ISSUE TITLE
Assigning tasks to programmers

### ISSUE THREAD

**Issue by maksimKorzh** [maksimKorzh/chessfuck#3]
MTP2148, for now the only programmers to assign tasks to are only you and I))), So what are your ideas?

**Comment by dmc75287** [comment]
@maksimKorzh I have the feeling that more will come onboard when news of this project gets around. We can announce it on r/brainfuck. If not, then we can work on it. I'm not in any hurry. Also it would be great if Daniel Christophani heard about this project. He's a grandmaster of brainfuck programring.

**Comment by maksimKorzh** [comment]
@dmc75287 I think that the announcement that has been already made is quite enough for now, I don't want to force people. Some may want to join after my tutorial on coding a chess program is released. I'm not in a hurry either, so let's work on our own for now and just see what happens.

**Comment by maksimKorzh** [comment]
@dmc75287 btw you can announce it at reddit if you like. There are too many of posts already)))

**Comment by dmc75287** [comment]
@maksimKorzh 
I agree, we can make some headway and then post updates on reddit, talkchess and maybe a few other related places.
